src/language/concurrency.md

```diff
@@ -144,80 +168,106 @@ void main() [!async!] {
 {% endprettify %}
 
 The `main()` function uses the `await` keyword in front of `_readFileAsync()`
-to let other Dart code (such as event handlers) use the CPU
-while native code (file I/O) executes.
-Using `await` also has the effect of
-converting the `Future<String>` returned by `_readFileAsync()` into a `String`.
-As a result, the `contents` variable has the implicit type `String`.
-
-{{site.alert.note}}
-  The `await` keyword works only in functions that
-  have `async` before the function body.
+to let other Dart code (such as event handlers) use the CPU while native code
+(file I/O) executes. Using `await` also has the effect of converting
+the `Future<String>` returned by `_readFileAsync()` into a `String`. As a
+result, the `contents` variable has the implicit type `String`.
+
+{{site.alert.note}} 
+The `await` keyword works only in functions that have `async` before the 
+function body. 
 {{site.alert.end}}
 
-As the following figure shows,
-the Dart code pauses while `readAsString()` executes non-Dart code,
-in either the Dart virtual machine (VM) or the operating system (OS).
-Once `readAsString()` returns a value, Dart code execution resumes.
+As the following figure shows, the Dart code pauses while `readAsString()`
+executes non-Dart code, in either the Dart virtual machine (VM) or the operating
+system (OS). Once `readAsString()` returns a value, Dart code execution resumes.
 
-![Flowchart-like figure showing app code executing from start to exit, waiting for native I/O in between](/assets/img/language/concurrency/basics-await.png)
+![Flowchart-like figure showing app code executing from start to exit, waiting 
+for native I/O in between](/assets/img/language/concurrency/basics-await.png)
 
-If you'd like to learn more about using `async`, `await`, and futures,
-visit the [asynchronous programming codelab][].
+### Stream syntax
```

Comment:
Same as the futures section, this section is not about syntax, it's about what streams are.